Tests/MaliciousSiteProtectionTests/MaliciousSiteProtectionUpdateManagerTests.swift

```diff
@@ -290,12 +290,12 @@ class MaliciousSiteProtectionUpdateManagerTests: XCTestCase {
         // Advance the clock by 1 seconds
         await self.clock.advance(by: .seconds(1))
         // expect to receive v.2 update for hashPrefixes
-        await fulfillment(of: [hashPrefixUpdateExpectations[1], hashPrefixSleepExpectations[1]], timeout: 1)
```

Comment:
@mallexxx I don't really know what this test is about, but it's now failing on the these 2 expectations for a timeout, If I increase the timeout to 2 and 3s the everything works, could you help me understand what's going on? Is increasing the timeout ok?

Reply:
no, increasing the real timeout isn‘t ok, mock clock is used here.
I‘ve updated the test now it should pass

Reply:
Thanks!